l10n/de_DE.js

```diff
@@ -70,6 +70,10 @@ OC.L10N.register(
     "Voice" : "Anruf",
     "Social network" : "Soziales Netzwerk",
     "Details" : "Details",
-    "Settings" : "Einstellungen"
+    "Settings" : "Einstellungen",
+    "Gender" : "Geschlecht",
+    "Male" : "männlich",
+    "Female" : "weiblich",
+    "intersex" : "intersexuell"
```

Comment:
thanks. do i understand [this](https://docs.nextcloud.com/server/11/developer_manual/core/translation.html#automated-synchronization-of-translations) right? I don't need to take care of translation? Your server is doing the scanning of new labels, pushes it to transifex, your crew will translate it and pushes it to github? All automatically? Thanks!